testing/environments/minimal/run.sh

```diff
@@ -37,16 +37,15 @@ aws ecr get-login-password --region us-west-2 \
  | docker login --username AWS --password-stdin 368821881613.dkr.ecr.us-west-2.amazonaws.com/confluentinc/
 
 # Download latest cp-zookeeper image.
-docker pull 368821881613.dkr.ecr.us-west-2.amazonaws.com/confluentinc/cp-zookeeper:master-latest
-
+docker pull 368821881613.dkr.ecr.us-west-2.amazonaws.com/confluentinc/cp-zookeeper:7.1.x-latest
```

Comment:
Same as below - how do we remember to flip this back to master for the pint merge? I wonder if (in a follow up PR) this could be parameterised so we don't have to remember each time we branch a release to go and update it.

Reply:
Yeah, we need a better solution for updating this at release cutting. Right now we just have to remember, which we never do. I will make sure to add a change to master when pint merging.